opm/autodiff/BlackoilWellModel_impl.hpp

```diff
@@ -660,45 +660,53 @@ namespace Opm {
     BlackoilWellModel<TypeTag>::
     getWellConvergence(const std::vector<Scalar>& B_avg) const
     {
-        ConvergenceReport report;
```

Comment:
There was already a ConvergenceReport struct defined in the WellInterface class, I chose a different name to avoid clashes during development. I'll change the name.